### ISSUE TITLE
Problem with goloc command in NPCs location command

### ISSUE THREAD

**Issue by larsbraunshausen** [Nutty101/NPC_Destinations2#43]
When I add a goloc command to location commands, I can't set the NPC ID within the location command. The "--npc ID" disappears when entered.

command:
/npcdest --npc 39 locaddcmd 0 nd goloc 1 --npc 39

added location command: 
npcdest goloc 1

**Comment by Nutty101** [comment]
Yup, it seems I replace both in the string. I will work on that.
